wpilib/wpilib/livewindow.py

```diff
@@ -22,45 +33,28 @@ class LiveWindow:
     """The public interface for putting sensors and
     actuators on the LiveWindow."""
 
-    sensors = set()
-    #actuators = set()
     components = {}
-    livewindowTable = None
-    statusTable = None
+    liveWindowTable = NetworkTablesInstance.getDefault().getTable("LiveWindow")
+    statusTable = liveWindowTable.getSubTable(".status")
+    enabledEntry = statusTable.getEntry("LW Enabled")
```

Comment:
nononononononononono

Reply:
what are the semantics of _reset? I assumed it was to recreate the initial state. What's the appropriate thing to do here?

Reply:
this is at the class level, not in any method defined — I'm pretty sure this will cause NT to be initialised

Reply:
The `reset` function is probably misnamed -- it probably should be called 'free'. The primary purpose of the reset function is to ensure that wpilib is correctly reinitialized during unit tests (all of the reset functions are called during test... teardown? or startup. Can't remember).

Reply:
And yeah, we shouldn't be initializing NT here, that potentially causes problems. We'll need to find a different way to initialize the class variable (maybe via a get function that initializes it if not present? Look to SmartDashboard, I think it does a similar thing).